### PR TITLE
Updated ReturnBookUI class for review

### PR DESCRIPTION
**Pull request by lukexjohnson**
- **ReturnBookUI:** White spacing, meaningful class names, capitilization of classes completed for review
- **Main:** Amended names of method calls to match that of ReturnBookUI
- **ReturnBookControl:**  Amended names of method calls to match that of ReturnBookUI

### CODE REVIEW

File: ITC205_BaseLine_Code/src/ReturnBookControl.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void setUI(ReturnBookUI ui) {
 			throw new RuntimeException("ReturnBookControl: cannot call setUI except in INITIALISED state");
 		}	
 		this.ui = ui;
-		ui.setState(ReturnBookUI.UI_STATE.READY);
+		ui.SetState(ReturnBookUI.UiState.READY);
 		state = CONTROL_STATE.READY;		
 	}
```
Comment (Millsy66) [inline ITC205_BaseLine_Code/src/ReturnBookControl.java:25]: D1 - delete blank line? - line 25

File: ITC205_BaseLine_Code/src/ReturnBookControl.java [hunk 2]
```diff
@@ -59,7 +59,7 @@ public void scanningComplete() {
 		if (!state.equals(CONTROL_STATE.READY)) {
 			throw new RuntimeException("ReturnBookControl: cannot call scanningComplete except in READY state");
 		}	
-		ui.setState(ReturnBookUI.UI_STATE.COMPLETED);		
+		ui.SetState(ReturnBookUI.UiState.COMPLETED);		
 	}
```
Comment (Millsy66) [inline ITC205_BaseLine_Code/src/ReturnBookControl.java:65]: D2 - delete blank line - line 65

File: ITC205_BaseLine_Code/src/ReturnBookControl.java (hunk 1 above)
Comment (lukexjohnson) [reply ITC205_BaseLine_Code/src/ReturnBookControl.java:25]: Will amend ReturnBookControl in alternate branch. This is the next branch to be amended.

File: ITC205_BaseLine_Code/src/ReturnBookControl.java (hunk 2 above)
Comment (lukexjohnson) [reply ITC205_BaseLine_Code/src/ReturnBookControl.java:65]: Will amend ReturnBookControl in alternate branch. This is the next branch to be amended.

**Pull request merged**